item-impl/src/main/java/com/example/auction/item/impl/ItemRepository.java

```diff
@@ -34,7 +34,6 @@
     @Inject
     public ItemRepository(CassandraSession session, ReadSide readSide) {
         this.session = session;
-        session.underlying().thenAccept(s -> registerCodec(s, new EnumNameCodec<>(ItemStatus.class)));
```

Comment:
TBH, I'm not sure when's the right moment to register this codec. When running several things on my laptop I could get errors claiming this codec was not available (race conditions). Also, I'm not 100% sure it needs to be on `prepareGlobal` or if it's safe to have all nodes on our cluster register the code again and again.